src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/PaymentsHandler.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.definition.ExceptionRecordFields;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.definition.YesNoFieldValues;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.servicebus.domains.payments.IPaymentsPublisher;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.servicebus.domains.payments.model.UpdatePaymentsCommand;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+
+import java.util.Objects;
+
+@Component
+public class PaymentsHandler {
+
+    private static final Logger log = LoggerFactory.getLogger(PaymentsHandler.class);
+
+    private final IPaymentsPublisher paymentsPublisher;
+
+    public PaymentsHandler(IPaymentsPublisher paymentsPublisher) {
+        this.paymentsPublisher = paymentsPublisher;
+    }
+
+    public void handlePayments(CaseDetails exceptionRecord, long newCaseId) {
```

Comment:
After extracting it from the 'context' of `CreateCaseCallbackService` this method name (and class name) sound **very** vague.

Reply:
will this class also handle creating the initial payment (when exception record is created)?

Reply:
Here we're essentially removing payments from exception record and adding them to newly created case.
So maybe something `movePayments` or `updatePayments`?

Reply:
movePaymentsToCase ?

Reply:
It actually sends UpdatePayments command to payments queue and is not in charge of any manipulations with payments so I think 'handleCasePayments' will be accurate enough